.github/workflows/deploy.yml

```diff
@@ -33,3 +46,45 @@ jobs:
       version_to_build: $(echo "${{ github.ref }}" | sed -e 's,.*/\(.*\),\1,')
       owner: ${{ github.repository_owner }}
       application: funding-service-design-account-store
+  copilot_build:
+    if: ${{ github.event.inputs.copilot }}
+    needs: [test_and_deploy]
+    concurrency: deploy-${{ inputs.environment || 'test' }}
```

Comment:
No harm whatsoever in this, but given it's an option above, surely it can't actually be blank, so there's no need for an extra default?

Reply:
It's an option on _dispatch_, as in, when we manually run this. But it also triggers on _push_ as well, which means in that case it will not have a value. This needs a minor refactor once we're doing other environments (aka main)